tests/unit/models/test_badge.py

```diff
@@ -0,0 +1,73 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+from databuilder.models.badge import Badge, BadgeMetadata
+
+from databuilder.models.neo4j_csv_serde import NODE_KEY, NODE_LABEL, \
+    RELATION_START_KEY, RELATION_START_LABEL, RELATION_END_KEY, \
+    RELATION_END_LABEL, RELATION_TYPE, RELATION_REVERSE_TYPE
+
+db = 'hive'
+SCHEMA = 'BASE'
+TABLE = 'TEST'
+CLUSTER = 'DEFAULT'
+badge1 = Badge('badge1', 'column')
+badge2 = Badge('badge2', 'column')
+
+
+class TestBadge(unittest.TestCase):
```

Comment:
after the verification is added , add a test to test both cases and verify the exception should be shown.